### PR TITLE
Sold out message

### PR DESCRIPTION
**Pull request by briancaine**
#193 and #177 

I don't have Chinese or Russian translations done for the strings I added though.

### CODE REVIEW

File: src/sender/dummy.go [hunk 1]
```diff
@@ -142,11 +155,17 @@ func (s *DummySender) IsTxConfirmed(txid string) *ConfirmResponse {
 	}
 }
 
-// Balance always returns 100 coins and 100 hours
 func (s *DummySender) Balance() (*cli.Balance, error) {
+
+	coin_str, err := droplet.ToString(s.coins)
```
Comment (gz-c) [inline src/sender/dummy.go]: Go uses camelCase

**Review by gz-c — CHANGES_REQUESTED** [review]
You have to put the same keys into the other translation files

File: src/sender/dummy.go (hunk 1 above)
Comment (briancaine) [reply src/sender/dummy.go]: https://github.com/skycoin/teller/pull/195/commits/ef381bb6c23ee701c3433d84a5712119242cc735

Fixed.

**Comment by briancaine** [comment]
@gz-c Added the keys with English transations.

**Review by gz-c — APPROVED** [review]

**Pull request merged**